plugins/Diagnostics/Diagnostic/DatabaseAbilitiesCheck.php

```diff
@@ -50,6 +50,13 @@ public function execute()
         $result->addItem($this->checkTemporaryTables());
         $result->addItem($this->checkTransactionLevel());
 
+        $databaseVersion = Db::fetchOne('SELECT VERSION();');
+
+        if (strpos(strtolower($databaseVersion), 'mariadb') !== false && Config\DatabaseConfig::getConfigValue('schema') !== 'Mariadb') {
```

Comment:
Should we have that check in the other direction (with `===` operators)?

Having MariaDB configured but connecting to MySQL would break the max execution time features. Could happen during the installation, as there is no safe guard against choosing that combination.

Reply:
You mean checking for `schema === 'Mysql'`? I did it on purpose that way, if we might add other schema later 🤔

Reply:
More like `databaseVersion !== x && configValue === x`, only saying "you have choosen poorly".

Having those checks if no `databaseVersion !== x && configValue !== x` was previously triggered should always give guidance for the correct one first, falling back to "please remove the config" if no suggestion is possible :thinking:

Reply:
The current check will only suggest to switch to Mariadb in config if the database version contains `mariadb` and it is not yet configured. In all other cases nothing will be shown.
Can we be sure that it is not a MariaDb server if the version does not contain MariaDb?
I'm actually not sure if we can fully rely on the returned database version. Will every MariaDb server contain `mariadb` in the returned version, or could that be hidden some way? 🤔

Reply:
Of course the version can be manipulated at will: https://mariadb.com/kb/en/server-system-variables/#version

We will never be sure, but if we only show it as a "please verify your setup" information, it may also not hurt. Could also be added once we have more databases available (and find a good wording for the message) and/or some user reports :+1:

Reply:
Ok. maybe we should rephrase it a bit and change it to informational instead of warning

Reply:
changed that.